SPECS/secilc/secilc.signatures.json

```diff
@@ -1,6 +1,5 @@
 {
  "Signatures": {
-  "secilc-2.9.tar.gz": "73a1806e33a669e23545da2d35d0e5038714721f6bf71974eaa533b3ebde61b2",
-  "0001-Allow-setting-arguments-to-xmlto-via-environmental-v.patch": "d41c968fd09b25fcdf341ffde2a3153982bb2fe329480048e58b9a39342a9962"
```

Comment:
is the patch removal intentional.  Ok if so, just double checking.

Reply:
We consider patches part of the local git repo. Since we don't download them we don't check the hashes on them.

Reply:
To be fair, we don't download a lot of other sources and still have signatures for them. I'm still not sure why we've decided to treat patches in a special way. But removing this for consistency seems OK for now.

---
In reply to: [504330356](https://github.com/microsoft/CBL-Mariner/pull/200#discussion_r504330356) [](ancestors = 504330356)